gateway/src/resty/sandbox.lua

```diff
@@ -0,0 +1,255 @@
+--- Sandbox
```

Comment:
Nice refactor :+1: 
I think this module does not contain any apicast-specific code so it could potentially be extracted into its own library, and that's why it's in the resty directory. However, there are some comments that mention apicast and policies. I think we should get rid of them. For example, the description of the module mentions APIcast, and the comment for `local env` mentions policies.

Reply:
Yep. I did not want to bother much with it now, but rather when it is extracted. Would that be ok?

Reply:
Sure.

Reply:
Done.